app/js/actions/patientAction.js

```diff
@@ -29,10 +33,22 @@ const addPatient = patient => ({
   patient,
 });
 
+const fetchPatientLabTestResults = patientUUID => ({
+  type: FETCH_PATIENT_LAB_TEST_RESULTS,
+  patientUUID,
+});
+
+const updatePatientInfo = patientInfo => ({
+  type: UPDATE_PATIENT_INFO,
```

Comment:
I'm not crazy about the name "update patient info" because it made me think this was actually a action that triggered updating the patient information in the DB.  I'm not sure exactly what the name should be however... what do you think about SET_PATIENT_DATA?